src/index.html

```diff
@@ -1,13 +1,402 @@
+                    <img
+                      class="place__photo"
+                      src="./images/photos/2.jpg"
+                      alt="Place 1"
+                    >
+                  </div>
+                </div>
+              </div>
+
+              <div class="place__info">
+                <a
+                  rel="noopener"
+                  target="_blank"
+                  href="https://goo.gl/maps/HWQWNWPpi8tKiWgc6"
+                  class="place__info-link"
+                >
+                  Marina Palms
+                </a>
+                / North Miami Beach, FL 33162
+              </div>
+            </div>
+          </div>
+
+          <div class="place gallery__place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-6">
+                  <div class="place__photo-container">
+                    <img
+                      class="place__photo"
+                      src="./images/photos/3.jpg"
+                      alt="Place 2"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-7-12">
+                  <div class="place__photo-container">
+                    <img
+                      class="place__photo"
+                      src="./images/photos/4.jpg"
+                      alt="Place 2"
+                    >
+                  </div>
+                </div>
+              </div>
+
+              <div class="place__info">
+                <a
+                  target="_blank"
+                  rel="noopener"
+                  href="https://goo.gl/maps/ANdpKkQdritGzFDJ6"
+                  class="place__info-link"
+                >
+                  Edition Residens
+                </a>
+                / Miami Beach, FL 33139
+              </div>
+            </div>
+          </div>
+
+          <div class="place gallery__place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4">
+                  <div class="place__photo-container">
+                    <img
+                      class="place__photo"
+                      src="./images/photos/5.jpg"
+                      alt="Place 3"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-5-12">
+                  <div class="place__photo-container">
+                    <img
+                      class="place__photo"
+                      src="./images/photos/6.jpg"
+                      alt="Place 3"
+                    >
+                  </div>
+                </div>
+              </div>
+
+              <div target="_blank" class="place__info">
+                <a href="https://goo.gl/maps/2PFsGTTNxyvs7rJk6" class="place__info-link">Faena House</a>
```

Comment:
Move target="_blank" to anchor link tag (```<a>```), it won't work if you put it on a div
```suggestion
              <div class="place__info">
                <a 
                href="https://goo.gl/maps/2PFsGTTNxyvs7rJk6" 
                class="place__info-link"
                target="_blank"
                >Faena House</a>
```